### ISSUE TITLE
fp16 training

### ISSUE THREAD

**Issue by bluestyle97** [Tangshitao/MVDiffusion#11]
Hi, thanks for the fantastic work and the codebase! Does this repo support fp16 training? I think I met some issues with the default fp16 training configuration of pytorch ligntening ...

**Comment by Tangshitao** [comment]
Fp16 cause nan loss in my experiment.

**Comment by y6216886** [comment]
Have you tried to clip large gradients to avoid nan?
